src/base/Package.zig

```diff
@@ -50,10 +50,11 @@ pub const Store = struct {
         primary_filename: []u8,
         all_primary_relative_paths: [][]u8,
         allocator: std.mem.Allocator,
-    ) Store {
-        const packages = List.init(allocator);
-        packages.append(Package{
+    ) !Store {
```

Comment:
Let's try not to propagate these errors in general by catching them at time of append with `exitOnOom`. I'll leave this comment and no more in the PR to avoid papercutting you to death.